public/index.html

```diff
@@ -21,55 +21,25 @@
         <!-- End Google Analytics -->
     </head>
     <body>
-        <div id="fb-root"></div>
-        <script>
-            (function(d, s, id) {
-                var js, fjs = d.getElementsByTagName(s)[0];
-                if (d.getElementById(id))
-                    return;
-                js = d.createElement(s);
-                js.id = id;
-                js.src = "//connect.facebook.net/en_US/all.js#xfbml=1&appId=322002764520105";
-                fjs.parentNode.insertBefore(js, fjs);
-            }(document, 'script', 'facebook-jssdk'));
-        </script>
         <div id="wrap">
             <div id="header">
                 <h1 id="title">LaTeX Online</h1>
                 <sup id="slogan">you give it a link, it gives you PDF</sup>
+                <a href="https://github.com/aslushnikov/latex-online" target="_blank" onClick="ga(‘send’, ‘event’, GitHubLink, ‘Click’, ‘MainPage’)"><img src="/GitHub.png" width="22" title="Fork Latex Online on GitHub"></a>
```

Comment:
![screen shot 2017-04-09 at 6 57 53 pm](https://cloud.githubusercontent.com/assets/24441206/24843760/9dc7533c-1d5a-11e7-8d47-e57f5e431d37.png)

this is how it looks like when in the top-right corner. I think it looks bad so that's why I committed it in the different place. 

But here is code to place it top-right corner, feel free to replace if you like this option more.

                    <a href="https://github.com/aslushnikov/latex-online" target="_blank" onClick="ga(‘send’, ‘event’, GitHubLink, ‘Click’, ‘MainPage’)"><img src="/GitHub.png" width="30" style="position: absolute; top: 0; right: 0; border: 0; margin:10px;" title="Fork Latex Online on GitHub"></a></p>